### DISCUSSION TITLE
[12789 도키도키 간식드리미] 왜 deque를 사용했을 때 더 느린가요?

### DISCUSSION THREAD

**Discussion by summerlunaa** [woowacourse-study/2022-lv3-algorithm-study#61] [문제 같이 풀어요~]
![image](https://user-images.githubusercontent.com/77425729/178106824-3c575391-a42e-444e-9284-4c73d06029a3.png)

[deque 사용X](https://www.acmicpc.net/source/45791055)
[deque 사용O](https://www.acmicpc.net/source/45790979)

리스트 양 끝의 값에 접근할 떄는 deque가 더 빠른 걸로 알고 있는데 왜 deque를 사용했을 때 더 느린가요?? 궁금합니다.

**Comment by asebn1** [comment]
그러게요...?

**Reply by SuyeonChoi** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ;;;;??

**Reply by summerlunaa** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by kth990303** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by Hyunta** [comment]
<img width="669" alt="Screen Shot 2022-07-10 at 4 12 46 PM" src="https://user-images.githubusercontent.com/75936123/178135040-3a9ce1c1-c15b-4175-9279-4bd0f1e8f195.png">

https://dev.to/v_it_aly/python-deque-vs-listwh-25i9
line.popleft() 에서는 deque가 성능이 좋은데 line[0], stack[-1] 여기서 까먹는게 아닐까요?
정확한건 왼손님한테 여쭤보면 확실히 알 수 있을 것 같아요!

**Reply by summerlunaa** [reply]
오 역시 아서~ 고맙습니다ㅎㅎㅎ
